shell/platform/tizen/tizen_renderer_ecore_wl2.cc

```diff
@@ -233,20 +234,20 @@ uintptr_t TizenRendererEcoreWl2::GetWindowId() {
 
 bool TizenRendererEcoreWl2::InitializeRenderer() {
   int32_t width, height;
-  if (!SetupDisplay(width, height)) {
-    FT_LOGE("SetupDisplay fail");
+  if (!SetupDisplay(&width, &height)) {
```

Comment:
if you use initial_geometry, maybe you don't really need this "width" and "height".

Reply:
As you can see in the documentation of `FlutterDesktopWindowProperties`, if the width and height are not provided by the app explicitly, the values are obtained from the current display just as usual.